iroha/src/asset.rs

```diff
@@ -64,13 +69,13 @@ pub mod isi {
         fn execute(&self, world_state_view: &mut WorldStateView) -> Result<(), String> {
             let asset = world_state_view
                 .account(&self.source_account_id)
-                .unwrap()
+                .expect("Source account not found")
```

Comment:
Why use `expect` if you can propagate to `Result`? Maybe `map_err` if you want to change it.

Reply:
You have this issue in both `execute` functions here.